File: VisualStudio_IDE/MmgGameApiCs/MmgGameApiCs/src/net/middlemind/MmgGameApiCs/MmgBase/MmgUpdateHandler.cs
using System;

namespace net.middlemind.MmgGameApiCs.MmgBase
{
    /// <summary>
    /// Class template for handling update events.
    /// Created by Middlemind Games 06/01/2005
    ///
    /// @author Victor G.Brusca
    /// </summary>
    [System.Diagnostics.CodeAnalysis.SuppressMessage("Style", "IDE1006:Naming Styles", Justification = "<Pending>")]
    public interface MmgUpdateHandler
    {
        /// <summary>
        /// Handle the incoming update event.
        /// </summary>
        /// <param name="obj">The update event object.</param>
        public void MmgHandleUpdate(Object obj);
    }
}

File: NetBeans_IDE/MmgGameApiJava/src/net/middlemind/MmgGameApiJava/MmgBase/MmgUpdateHandler.java
package net.middlemind.MmgGameApiJava.MmgBase;

/**
 * Class template for handling update events.
 * Created by Middlemind Games 06/01/2005
 *
 * @author Victor G. Brusca
 */
public interface MmgUpdateHandler {

    /**
     * Handle the incoming update event.
     *
     * @param obj   The update event object.
     */
    public void MmgHandleUpdate(Object obj);
}